prep release: v1.57.1 (#6214)
apollographql/router: CHANGELOG.md

This project adheres to [Semantic Versioning v2.0.0](https://semver.org/spec/v2.0.0.html).

# [1.57.1] - 2024-10-31

## 🐛 Fixes

### Progressive override: fix query planner cache warmup ([PR #6108](https://github.com/apollographql/router/pull/6108))

> This fixes an issue in progressive override where the override labels would not be transmitted to the query planner during cache warmup, resulting in queries correctly using the overridden fields at first, but after an update, would revert to non overridden fields, and could not recover.

Reviewer: minor edit to make it maybe more grokkable, in my mind atleast
```suggestion
This fixes an issue in progressive override where the override labels were not transmitted to the query planner during cache warmup. Queries were correctly using the overridden fields at first, but after an update, reverted to non overridden fields, and could not recover.
```
